ResourceManager/LinkFactory.cs

```diff
@@ -48,7 +48,7 @@ public string CreateLink(string? caption, string uri)
         private string AddLink(string? caption, string uri)
         {
             string htmlUri = WebUtility.HtmlEncode(uri);
-            string rtfLinkText = caption + "#" + htmlUri;
+            string rtfLinkText = caption;
```

Comment:
💭 Oh interesting. Is this going to change any observable behavior?

Reply:
We had to do a bit of work in dotnet/winforms to address issues in this area. 
E.g. https://github.com/dotnet/winforms/issues/1644 and https://github.com/dotnet/winforms/issues/3632

I've done some testing in the app, and - no, no observable change in behaviours. `RichTextBox.DetectUrls = false` does the magic (https://github.com/dotnet/winforms/issues/3632#issuecomment-752543714)